### ISSUE TITLE
Header Stylesheet

### ISSUE THREAD

**Issue by rodloboz** [tomgauth/routine.io#10]
@tomgauth 

I have added a few classes from the components/_button.scss to the header button to make the style uniform in terms of buttons (created a few extra classes to accommodate the previous style).

Couldn't help notice that the _header.scss is full of CSS code, most not being used in the header.

**Comment by tomgauth** [comment]
Yeah you're right. I copied pasted a whole theme just to get this home page done quickly.

**Comment by rodloboz** [comment]
I do that all the time as well, but if we're not careful we end up with a code that's harder for us to read through and, although not really important in a prototype, it can lead to poor performance as the larger the CSS the slower the pages load
